enhancements/monitoring/replace-prometheus-adapter-with-metrics-server.md

```diff
@@ -0,0 +1,302 @@
+
+1. **Better Accuracy:**  By default, the Metrics Server takes into account the timestamps of metric samples, allowing for accurate tracking of resource utilization over time. This means you can rely on the Metrics Server to provide a more precise understanding of how resources are being utilized by your applications.
+
+1. **Scability:** Starting from `v0.5.0` Metrics Server comes with default resource requests that guarantee good performance for most cluster configurations and it continues to do [scale testing](https://github.com/kubernetes-sigs/metrics-server/blob/077a462cd660beef705afb0a4a14a707ef39c4f2/FAQ.md#how-large-can-clusters-be) which is not the case for Prometheus Adapter.
+
+### User Stories
+
+#### Story 1
+
+* In order to provide [optional built-in monitoring](https://issues.redhat.com/browse/MON-3152) feature we need to provide a way for customers to do HPA other than through Prometheus based Prometheus Adapter.
+
+#### Story 2
+
+* As a cluster admin, I seek a lightweight and efficient solution with improved accuracy for monitoring resource metrics within my cluster.
+
+#### Story 3
+
+* As a cluster admin, I want a simplified monitoring architecture in OpenShift, so that I can reduce the complexity and maintenance overhead of the monitoring system.
+
+### Goals
+
+* Replace Prometheus Adapter with Metrics Server while maintaining feature parity with the existing stack 
+* Ensure no performance regression occurs when replacing Prometheus Adapter with Metrics Server
+
+### Non-Goals
+
+* Allow optional cluster monitoring stack
+* Moving `openshift-monitoring/cluster-monitoring-config` config-map to a Custom Resource
+
+## Proposal
+
+The proposal is to replace the Prometheus Adapter with the Metrics Server for monitoring resource metrics in OpenShift.
+
+### Workflow Description
+
+The workflow for using the Metrics Server in OpenShift for monitoring resource metrics would involve the following steps:
+
+1. User sets `enabled: true` in the ConfigMap `openshift-monitoring/cluster-monitoring-config` under `metricsServer` section
+
+    ```yaml
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: cluster-monitoring-config
+      namespace: openshift-monitoring
+    data:
+      config.yaml: |
+        metricsServer:
+          enabled: true
+    ```
+
+With this configuration, the cluster monitoring operator should:
+  1. Deploy the metrics-server resources in the `openshift-monitoring` namespace
+  1. Verify that the `metrics-server` service is functional by querying the `/-/readyz` endpoint
+  1. Wait for a couple of minutes ( 2 to 3 scrape cycles) to have enough data populated in `metrics-server` in memory 
```

Comment:
I'd suggest waiting for 5 times the scrape interval to account for potentially failed scrapes.
```suggestion
  1. Wait for ` 5 x metrics-server's scrape interval`  to have enough data populated in `metrics-server` in memory 
```